### DISCUSSION TITLE
Small thanks from somewhere far

### DISCUSSION THREAD

**Discussion by bennysway** [jendrikseipp/rednotebook#743] [Show and tell]
![Screenshot_2024-01-17-22-04-43-541_com google android keep-edit](https://github.com/jendrikseipp/rednotebook/assets/23507694/bbab335f-1c26-43b2-bcd0-b585f4f992de)

I just want to thank you guys for making this app. I met it when I was still in Zimbabwe, still a kid and loving every thing Linux. It helped me a lot. 

Sometime, I don't know when, I lost devices and the passion for Linux but deep down I could stop writing, that's why there is a screenshot of Google Keep of my university days in Poland. Now I'm a senior android dev (engineer soon if I can make it) and I hope I can chip in some time to make a simple android version. Because this app was simple enough to make me never stop writing.

Thank you Team. 
Ben

**Comment by jendrikseipp** [comment]
Thanks for the kind words! An Android version would certainly be cool! It certainly has been requested lots of times (e.g., #165).
